packages/workflows/src/handlers/remove-inventory-items.ts

```diff
@@ -1,31 +1,26 @@
-import { InventoryItemDTO, MedusaContainer } from "@medusajs/types"
+import { InventoryItemDTO } from "@medusajs/types"
 
 import { InputAlias } from "../definitions"
-import { WorkflowArguments } from "../helper"
+import { PipelineHandlerResult, WorkflowArguments } from "../helper"
 
-export async function removeInventoryItems({
+export async function removeInventoryItems<T = void[]>({
   container,
   data,
 }: WorkflowArguments & {
   data: {
     [InputAlias.InventoryItems]: InventoryItemDTO
   }[]
-}) {
+}): Promise<PipelineHandlerResult<T>> {
   const manager = container.resolve("manager")
   const inventoryService = container.resolve("inventoryService")
   const context = { transactionManager: manager }
 
-  const value = await Promise.all(
+  return (await Promise.all(
     data.map(async ({ [InputAlias.InventoryItems]: inventoryItem }) => {
       return await inventoryService!.deleteInventoryItem(
```

Comment:
can't wait to have this refactored to delete in bulk 😆

Reply:
yeah definately 😂